project.html

```diff
@@ -0,0 +1,92 @@
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+</head>
+<body>
+
+
+<nav class="navbar navbar-expand-md bg-dark navbar-dark">
+  <a class="navbar-brand" href="index.html">Mon Site Web</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="collapsibleNavbar">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="index.html">Home</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="about.html">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="project.html">Project</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="contact.html">Contact</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+<main class="container">
+    <h1>Mes Projets</h1>
+    <div class="row">
+        <div class="col-12 project-container">
+            <h2>Jeu de cartes</h2>
+            <div class="row">
+                <div class="col-md-4">
+                    <img class="project-img" src="images/jeuDeCarte.jpg" alt="Jeu de cartes">
+                </div>
+                <div class="col-md-8">
+                    <p>Un jeu de mémoire réalisé en HTML, CSS et JavaScript. Le joueur doit saisir le nombre de paires qu'il y aura dans le jeu et saisir son nom. Un timer de 5 minutes ce lance au début de la parti pendant lequel le joueur doit compléter le jeu. Un message apparait si le joueur parvient à terminer le jeu avec le temps.</p>
+                </div>
+            </div>
+        </div>
+        <div class="col-12 project-container">
+            <h2>Projet Figma</h2>
+            <div class="row">
+                <div class="col-md-4">
+                    <img class="project-img" src="images/golf.jpg" alt="Projet Figma">
+                </div>
+                <div class="col-md-8">
+                    <p>Un projet de design réalisé avec Figma. Il s'agissait de concevoir l'interface utilisateur d'un site web de vente de bâton de golf pour joueurs exprimentés. Le visuel et l'interface respecte la clientèle visée en mettant de l'avant la qualité et la performance des produits proposés. </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-12 project-container">
+            <h2>Formulaire de voyage</h2>
+            <div class="row">
+                <div class="col-md-4">
+                    <img class="project-img" src="images/voyage.jpg" alt="Formulaire de voyage">
+                </div>
+                <div class="col-md-8">
+                    <p>Un formulaire de réservation de voyage réalisé en HTML et CSS. L'objectif était d'être capable de réaliser un formulaire.</p>
```

Comment:
On mobile, your footer is "eating" the content here: 

![image](https://user-images.githubusercontent.com/6362750/235274079-2c2d5531-75e3-4444-a54f-c72175dd36cd.png)